app/screens/channel_info/channel_info_header.js

```diff
@@ -157,6 +161,18 @@ export default class ChannelInfoHeader extends React.PureComponent {
             android: style.detail,
         });
 
+        const customStatusExpiry = customStatus?.emoji && customStatus?.duration !== CustomStatusDuration.DONT_CLEAR ?
+            (
+                <Text style={style.customStatusExpiry}>
+                    <CustomStatusExpiry
+                        time={customStatus.expires_at}
+                        theme={theme}
+                        textStyles={style.customStatusExpiry}
+                        showPrefix={true}
+                    />
+                </Text>
+            ) : null;
```

Comment:
this logic should be moved to L#218. It would be even better to create a component that encapsulates all custom status related stuff to be mounted a line 200